ee/watchdog/controller_windows.go

```diff
@@ -410,6 +411,31 @@ func installWatchdogTask(identifier, configFilePath string) error {
 		return fmt.Errorf("setting event trigger delay: %w", err)
 	}
 
+	// add a second trigger using the same event subscription, but without a delay.
+	// this way we will fire immediately upon wake up, and again a minute later in case
+	// things were too busy during wake
+	noDelayTriggerResp, err := oleutil.CallMethod(triggers, "Create", uint(0)) // 0=TASK_TRIGGER_EVENT
+	if err != nil {
+		log.Fatalf("encountered error creating trigger: %s", err.Error())
```

Comment:
I don't think we want a `Fatalf` here -- `return fmt.Errorf`?

Reply:
oop good catch! thank you